detect_secrets/plugins/aws.py

```diff
@@ -24,13 +24,16 @@ class AWSKeyDetector(RegexBasedDetector):
 
     denylist = (
         re.compile(r'AKIA[0-9A-Z]{16}'),
+        re.compile(r'aws.{0,20}?[\'\"]([0-9a-zA-Z/+]{40})[\'\"]'),
     )
 
     def verify(       # type: ignore[override]  # noqa: F821
         self,
         secret: str,
         context: CodeSnippet,
     ) -> VerifiedResult:
+        if not self.denylist[0].match(secret):
```

Comment:
To verify the secret, a secret like `AKIA[0-9A-Z]{16}` is needed, so the verify method has been updated to return `UNVERIFIED` if a secret like `aws.{0,20}?[\'\"]([0-9a-zA-Z/+]{40})[\'\"]` is found.